source/energy/policy.rst

```diff
@@ -0,0 +1,37 @@
+   1. Emission targets
+   2. Energy shares
+   3. Capacity or generation targets
+   4. Macro-economic targets
+A detailed description of the methodological implementation of the NDCs in the global energy model, along with an extensive list of the energy related targets considered can be found in Rogelj et al. (2017) :cite:`rogelj_indc_2017`. Additional policies implemented in the model can also be found in ('what reference for the CD_Links related policies?`)
+
+Emission targets
+----------------
+Country specific emission reduction targets are specified either in relation to a reference emission trajectory (in the form of a baseline or business as usual scenario (BAU)). The targets themselves are expressed as either (1) absolute reduction, (2) a percentage reduction or (3) intensity reductions e.g. emissions per GDP or per capita. In order to account for these different reduction targets in the global energy model, the targets are translated so that a regionally specific upper bound on emissions can be formulated. If not further specified, emission constraints are assumed to apply to all sectors and all gases, i.e. total GHGs.
+
+Energy shares
+-------------
+Energy share targets refer to any target which aims to provide a specific energy level (e.g. primary, secondary or final energy) through a specific sub-set of energy forms.  The five different forms in which these are formulated in the NDCs are: (1) renewable energy as share of total primary energy, (2) non-fossil energy forms as share of total primary energy, (3) renewable energy as a share of total electricity generation, (4) non-fossil energy as a share of total electricity generation, (5) renewable energy as a form of final energy.  All of these share constraint variants can be implemented in the model using the following `mathematical formulation <https://message.iiasa.ac.at/en/stable/model/MESSAGE/model_core.html#constraints-on-shares-of-technologies-and-commodities>`_. In order to be able to implement these for aggregate regions, it is necessary to harmonize these to single type of share constraint, so that their effects are considered cumulatively within a region. All variants are therefore harmonized to either the share type specified by the largest country, in terms of share of energy within a region, or the most frequently specified type within a region.
+Separately biofuel shares are implemented specifically for the transport sector.
+
+
+Capacity and generation targets
+-------------------------------
+Some NDCs specify capacity installation targets, e.g. for planned power plants which will be operational by a certain year.  Others specify that a given energy commodity will come from a specific source, for example a certain amount of electricity will stem form a specific intermittent renewable source or nuclear. These targets types are implemented in the model as lower bounds on generation.
+
+
+Macro-economic targets
```

Comment:
The description seems empty here.

Reply:
@volker-krey maybe you can provide input on this section, explaining how some of the policies were implemented by adjusting macro related parameters?

Reply:
@volker-krey a gentle reminder here.